src/lib/list/_list-theme.scss

```diff
@@ -46,7 +46,8 @@
     }
 
     .mat-subheader {
-      @include mat-typography-level-to-styles($config, body-2);
+      font: mat-font-weight($config, body-2) mat-font-size($config, body-2)
```

Comment:
The `font` property is used in this file for the dense mode as well ([see here](https://github.com/DevVersion/material2/blame/110c54cab6616712fc8b8f91cff667af5f1799f8/src/lib/list/_list-theme.scss#L62)).

I'm fine changing it for both if desired.